src/phpDocumentor/Configuration/ConfigurationFactory.php

```diff
@@ -13,15 +13,16 @@
 
 namespace phpDocumentor\Configuration;
 
-use League\Uri\Uri;
+use League\Uri\Contracts\UriInterface;
 use phpDocumentor\Configuration\Exception\InvalidConfigPathException;
+use phpDocumentor\UriFactory;
 use function file_exists;
 use function sprintf;
 
 /**
  * The ConfigurationFactory converts the configuration xml from a Uri into an array.
  */
-final class ConfigurationFactory
+/*final*/ class ConfigurationFactory
```

Comment:
Commented final? Shouldn't we remove the final or fix the tests?

Reply:
Test needs to be fixed, but that needs to be done in a second pr. Because it feels odd to introduce an interface to be able to mock this class, without having it properly shaped. We still do need to merge the symfonyConfigFactory into this class.